eml_parser/eml_parser.py

```diff
@@ -669,37 +670,61 @@ def string_sliding_window_loop(body: str, slice_step: int = 500, max_distance: i
                 ptr_start = ptr_end
 
     @staticmethod
-    def get_uri_ondata(body: str) -> typing.List[str]:
+    def get_uri_ondata(body: str, include_href: bool = True, email_force_tld: bool = False) -> typing.List[str]:
         """Function for extracting URLs from the input string.
 
         Args:
             body (str): Text input which should be searched for URLs.
+            include_href (bool): Include potential URLs in HREFs matching non-simple regular expressions
+            email_force_tld (bool): Exclude common file extensions that are not valid TLDs
```

Comment:
Could you please correct the description:
**file** -> domain

Reply:
Modified to better describe the filtering out invalid TLDs based on common file extensions.